src/main/java/au/com/rayh/XCodeBuilder.java

```diff
@@ -276,6 +276,7 @@ public boolean perform(AbstractBuild build, Launcher launcher, BuildListener lis
         String ipaOutputDirectory = envs.expand(this.ipaOutputDirectory);
         String bundleID = envs.expand(this.bundleID);
         String bundleIDInfoPlistPath = envs.expand(this.bundleIDInfoPlistPath);
+        String ipaManifestPlistUrl = envs.expand(this.ipaManifestPlistUrl);
```

Comment:
I've noticed today other variables are missing expansion. I should review the whole code at the same time as this pull request.